Source/Cells/Text/FORMTextFieldCell.m

```diff
@@ -273,7 +273,7 @@ - (CGRect)textFieldFrame {
 }
 
 - (void)showTooltip {
-    if (self.field.info && self.showTooltips) {
+    if (self.field.info && [self.field.info length] > 0 && self.showTooltips) {
```

Comment:
Looks good to me.
You may want to use `self.field.info.length` here as well to avoid `[]`